### PR TITLE
Adds biweekly recurring option

### PR DESCRIPTION
**Pull request by abea**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -78,11 +82,34 @@ module.exports = {
...
         def: 1
       }
-    ].concat(options.addFields || []);
+    ];
+
+    if (options.increment) {
+      eventSchema.push(
+        {
+          name: 'repeatIncrement',
+          label: 'Repetition increment',
```
Comment (boutell) [inline index.js]: This doesn't seem like friendly UX. Shouldn't it be more like:

Daily
Weekly
Every Other Week
Monthly
Every Other Month

cc @carlykaras

Comment (abea) [reply index.js]: I'd definitely like more thoughts on that. I didn't know whether to err towards flexibility or ease.

Comment (boutell) [reply index.js]: Then there's the UX of "every other week" itself. In my experience it doesn't work out well (people lose track in their heads) and "first and third Wednesdays" works a lot better, which makes me wish we supported that too (: Probably overkill just now though.

For fun, this is what I did on Salsadelphia, a perpetual calendar of Philly salsa events that is still perpetuating, despite my absence from the scene, as folks can just start editing if they are willing to log in via Facebook once and get moderated on their first edit:

Screen Shot 2019-04-02 at 3 41 08 PM

Everyone seems to figure this out right away. But, I had the luxury of leaving stuff out. I don't like "every other week" and knew events scheduled that way would just tank anyway, so I did not implement it. We don't have that kind of clout here.

Comment (carlykaras) [reply index.js]: I'd lean on flexibility more, but agree that the "every other week" can get dicey - maybe we should wait on the alternately recurring (every other week/every other month) until we have a more comprehensive approach.

Comment (abea) [reply index.js]: @carlykaras I'm not sure I understand. Are you advocating no change until a more full review can happen? I'll need to implement this on the specific project on the project level instead if so.

**Pull request closed without merging**